src/msg/Messenger.cc

```diff
@@ -30,14 +29,7 @@ Messenger *Messenger::create(CephContext *cct, const string &type,
 {
   int r = -1;
   if (type == "random") {
-    static std::random_device seed;
-    static std::default_random_engine random_engine(seed());
-
-    static std::mutex random_lock;
-    std::lock_guard<std::mutex> lock(random_lock);
-
-    std::uniform_int_distribution<> dis(0, 1);
-    r = dis(random_engine);
+    r = ceph::util::generate_random_number(0, 1);
   }
```

Comment:
The power of libraries!

Reply:
Brought a tear to my eye, it did!

Reply:
Mind to rename to `gen_rand_number`? ，to make consistent with rgw  naming style (like https://github.com/ceph/ceph/blob/master/src/rgw/rgw_common.cc#L830)